code/go/0chain.net/core/transaction/http.go

```diff
@@ -127,11 +127,11 @@ func VerifyTransaction(txnHash string, chain *chain.Chain) (*Transaction, error)
 	// 		Logger.Error("Error getting transaction confirmation", zap.Any("error", err))
 	// 		numSharders--
 	// 	} else {
-	// 		if response.StatusCode != 200 {
+	// 		if res.StatusCode != 200 {
```

Comment:
Why is there this lump of commented out code? If you must do someting delete it; to prevent others from being destracted by it.